msg/service.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/miekg/dns"
 )
 
+var PathPrefix string = "skydns"
```

Comment:
All look good. But can you add some docs to this export var? I.e. along the lines of the docs you added to skydns it self.